regression/cbmc/CMakeLists.txt

```diff
@@ -24,6 +24,14 @@ add_test_pl_profile(
     "CORE"
 )
 
+# If `-I` (include flag) is passed, test.pl will run only the tests matching the label following it.
+add_test_pl_profile(
+  "cbmc-new-smt-backend"
+  "$<TARGET_FILE:cbmc> --incremental-smt2-solver 'z3 --smt2 -in' --slice-formula"
+  "-I;new-smt-backend"
```

Comment:
We should add `-s;new-smt-backend` to this line so that separate log files get saved, rather than clobbering the log files from the standard run.

Reply:
Done.